glossary.yml

```
@@ -3100,6 +3100,28 @@
       de données, ni ne supporte l'ajout de commentaires.
 
 
+- slug: jupyter
+  ref:
+    - jupyter_notebook
+  en:
+    term: "Jupyter"
+    def: >
+      Project Jupyter is a non-profit open-source project that was born out of the IPython Project
+      in 2014 as it evolved to support interactive [data science](#data_science) and scientific computing
+      in many different programming languages.
+
+
+- slug: jupyter_notebook
+  ref:
+    - computational_notebook
+    - jupyter
+  en:
+    term: "Jupyter Notebook"
+    def: >
+      The Jupyter Notebook is an open-source web-based [computational notebook](#computational_notebook)
+      that allows you to write and share live code, equations, visualizations, and narrative text.
```

```suggestion
      that allows the user to write and share live code, equations, visualisations, and narrative text.
```
For future reference: British spelling is preferred in The Carpentries.